Client/SizeFilterConsumer.py

```diff
@@ -20,16 +21,25 @@ def callback(ch, method, properties, body):
         body = json.loads(body)
         print(body)
         params = body["params"]
-        if params["unit"] == "kb":
-                if "threshold" in params.keys():
-                    threshold = float(params["threshold"])
-                else:
-                    threshold = 0
 
-                res = filter_by_KB(paths = body["paths"], reference = float(params["kb"]), comparator = params["comparator"], threshold = threshold)
-                result = ResultResponse(200, res, SENDER)
+        if "threshold" in params.keys():
+                threshold = float(params["threshold"])
         else:
-            result = ResultResponse(501, [], SENDER)
+            threshold = 0
+        try:
+            if params["unit"] == "kb":
+                    res = filter_by_KB(paths = body["paths"], reference = float(params["kb"]), comparator = params["comparator"], threshold = threshold)
+                    result = ResultResponse(200, res, SENDER)
+            elif params["unit"] == "pixels":
+                res = filter_by_pixels(paths = body["paths"], reference = params["pixels"], comparator = params["comparator"], threshold = threshold)
+                result = ResultResponse(200, res, SENDER)
+            else:
+                result = ResultResponse(501, [], SENDER)
+        except Exception as e:
+            result = ResultResponse(404, [], SENDER)
```

Comment:
Since we don't really have any error message (maybe we should think about it), please try to use proper HTTP codes - 404 is not found and this looks like an internal server error (500). Would a good fix to go with the physical size PR in the future